components/brave_federated_learning/brave_operational_profiling.h

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_FEDERATED_LEARNING_BRAVE_OPERATIONAL_PROFILING_H_
+#define BRAVE_COMPONENTS_BRAVE_FEDERATED_LEARNING_BRAVE_OPERATIONAL_PROFILING_H_
+
+#include <memory>
+#include <string>
+
+#include "base/time/time.h"
+#include "base/timer/timer.h"
+#include "components/prefs/pref_service.h"
+#include "url/gurl.h"
+
+class PrefRegistrySimple;
+
+namespace network {
+
+class SharedURLLoaderFactory;
+class SimpleURLLoader;
+
+}  // namespace network
+
+namespace brave {
+
+class BraveOperationalProfiling {
+ public:
+  explicit BraveOperationalProfiling(
+      PrefService* pref_service,
+      scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory);
+  ~BraveOperationalProfiling();
```

Comment:
pls explicitly `= delete;` copy ctor and assignment operator. Could be useful to recheck Google and Chromium C++ style guides

Reply:
👍